src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBroker.cs

```diff
@@ -27,6 +27,9 @@ internal class iOSBroker : NSObject, IBroker
 
         private readonly ICoreLogger _logger;
         private readonly ICryptographyManager _cryptoManager;
+        private string BrokerRequestNonce { get; set; }
```

Comment:
You don't need properties for these.

Reply:
i was trying to access this elsewhere, but didn't remove the properties later. thx.

---
In reply to: [323183904](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1383#discussion_r323183904) [](ancestors = 323183904)